test/runtests.jl

```diff
@@ -1,12 +1,18 @@
 using ProximalOperators
 using ProximalAlgorithms
+using AbstractOperators
 using AbstractOperators.BlockArrays
-using Base.Test
+using LinearAlgebra
+using SparseArrays
+using FFTW
+using Random
+using Test
+using DelimitedFiles
 
 @testset "ProximalAlgorithms" begin
 
 @testset "Utilities" begin
-    include("test_block.jl")
+    include("test_block.jl")  # remove?
```

Comment:
not sure why we had these test, I believe they could be removed. @lostella don't you think?

Reply:
Let’s leave it as it is

Reply:
but all of the tests of 'test_block' are already in [AbstractOperators](https://github.com/kul-forbes/AbstractOperators.jl/blob/master/test/test_block.jl) and these test do not cover any code of 'ProximalAlgorithms' itself...